bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
+            'trigger_animation': self.run_animation
         }
 
+        self.trigger_procs = {}
+        self.triggers = {}
+        for t in triggers:
+            typename = t['typename']
```

Comment:
Yeah... I debated about that. I'm kinda keen on this hard failing because they may not see the message and then wonder why their trigger isn't happening. But I can certainly improve the hard fail error message.